.github/workflows/go.yml

```diff
@@ -1,14 +1,11 @@
 name: Go
 
 on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+  [push]
```

Comment:
NIT - not a blocker, but we sure we want this on every push?  Maybe just on PRs?  Every push might eat through GH actions time, although I don't know what our limit is these days organizationally

Reply:
I'm cool with that. I'll change it to prs to main & our hackathon branch